### PR TITLE
changed file

### PR DESCRIPTION
**Pull request by IgorPushnov**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+const pathFiles = fs.readdirSync(folderPath).map(fileName => {
+  return path.join(folderPath, fileName);
+});
+
+const textsForAnalysis = [];
+pathFiles.forEach(function (entry) {
+  const textExample = fs.readFileSync(entry, 'utf-8');
+  textsForAnalysis.push(textExample);
+  return textsForAnalysis;
+});
```
Comment (shnax0210) [inline main.js]: Please place all code related to files reading to single function, something like:

```
function readFiles(folder) {
  ...
}
```

File: main.js [hunk 2]
```diff
@@ -1,9 +1,17 @@
...
+const path = require('path');
 
-const textExample1 = fs.readFileSync('testData/cat.html', 'utf-8');
-const textExample2 = fs.readFileSync('testData/dog.css', 'utf-8');
-const textExample3 = fs.readFileSync('testData/world.txt', 'utf-8');
-const textsForAnalysis = new Array(textExample1, textExample2, textExample3);
+const folderPath = 'testData';
+const pathFiles = fs.readdirSync(folderPath).map(fileName => {
+  return path.join(folderPath, fileName);
+});
```
Comment (shnax0210) [inline main.js]: There is no need `{}` symbols and `return` statement in one line arrow function, so more short variant will be:

```
const pathFiles = fs.readdirSync(folderPath).map(fileName => path.join(folderPath, fileName););
```

File: main.js [hunk 3]
```diff
@@ -1,9 +1,17 @@
...
+
+const textsForAnalysis = [];
```
Comment (shnax0210) [inline main.js]: `textsForAnalysis` population can be done more short in next way:

```
const textsForAnalysis = pathFiles.map(pathToFile => fs.readFileSync(pathToFile, 'utf-8'));
```

File: main.js [hunk 4]
```diff
@@ -1,9 +1,17 @@
...
+pathFiles.forEach(function (entry) {
```
Comment (shnax0210) [inline main.js]: `entry` is not informative name

File: main.js (hunk 1 above)
Comment (IgorPushnov) [reply main.js]: changes made

File: main.js (hunk 2 above)
Comment (IgorPushnov) [reply main.js]: changes made

File: main.js (hunk 3 above)
Comment (IgorPushnov) [reply main.js]: changes made

File: main.js (hunk 4 above)
Comment (IgorPushnov) [reply main.js]: changes made

File: main.js [hunk 5]
```diff
@@ -1,9 +1,18 @@
...
+const pathFiles = fs.readdirSync(folderPath).map(fileName => path.join(folderPath, fileName));
+const textsForAnalysis = pathFiles.map(pathToFile => fs.readFileSync(pathToFile, 'utf-8'));
+
+function readFiles(folder) {
+  folder.forEach(function (arrayElement) {
+    const textExample = fs.readFileSync(arrayElement, 'utf-8');
+    textsForAnalysis.push(textExample);
+    return textsForAnalysis;
+  });
+}
```
Comment (shnax0210) [inline main.js:14]: Please have next code here:

```
function readFiles(folderPath) {
  const pathFiles = fs.readdirSync(folderPath).map(fileName => path.join(folderPath, fileName));
  return pathFiles.map(pathToFile => fs.readFileSync(pathToFile, 'utf-8'));
}
```

and then modify last line in the file:

```
console.log(calculateWordsStatistic(readFiles('testData')));
```

File: main.js (hunk 4 above)
Comment (IgorPushnov) [reply main.js]: changes made

File: main.js (hunk 5 above)
Comment (IgorPushnov) [reply main.js:14]: changes made

**Pull request merged**